client/extensions/tga-sign-off/src/components/fields/editor.tsx

```diff
@@ -33,37 +74,93 @@ export class UserSignOffField extends React.Component<IEditorProps> {
                 item_id: this.props.item._id,
                 authors: authorIds,
             }
+        }).then(() => {
+            notify.success(gettext('Sign off request email(s) sent'));
+        }, (error) => {
+            notify.error(gettext('Failed to send sign off request email(s). {{ error }}', error));
         });
     }
 
-    removeSignOff() {
-        const {confirm} = superdesk.ui;
+    sendSingleSignOff(authorId: IUser['_id']) {
+        const {notify} = superdesk.ui;
         const {gettext} = superdesk.localization;
 
-        confirm(gettext('Are you sure you want to remove this publishing sign off?'), gettext('Remove publishing sign off'))
-            .then((response) => {
-                if (response) {
-                    superdesk.entities.article.patch(this.props.item, {
-                        extra: {
-                            ...(this.props.item.extra ?? {}),
-                            publish_sign_off: {},
+        superdesk.httpRequestJsonLocal({
```

Comment:
this part seems pretty much the same like in `sendSignOff` only authors param is different, would be good to avoid duplication